### DISCUSSION TITLE
How to enable Experimental jvm metrics after version 2.0.

### DISCUSSION THREAD

**Discussion by zhangjiabin1010** [open-telemetry/opentelemetry-java-instrumentation#11230] [Q&A] [answered]
I am here
https://opentelemetry.io/docs/specs/semconv/runtime/jvm-metrics/ I didn’t see how to configure it.

I need indicators like this:
jvm.memory.init
jvm.system.cpu.utilization
jvm.buffer.memory.usage ...

**Comment by laurit** [comment, marked as answer]
See https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/runtime-telemetry/README.md
